packages/react-i18n/src/manager.ts

```diff
@@ -32,15 +32,15 @@ export default class Manager {
     initialTranslations: ExtractedTranslations = {},
   ) {
     for (const [id, translation] of Object.entries(initialTranslations)) {
-      this.translations.set(id, translation);
+      this.translations.set(id, translation || undefined);
```

Comment:
Is this really needed?

Reply:
Yes, `this.translations` is `TranslationDictonary | undefined` so `null` is not a valid value.

Do you think using
```javascript
this.translations.set(id, translation === null ? undefined : translation);
```
instead be more readable?

Reply:
Oh right I missed that `initialTranslations` was an `ExtractedTranslations`. `|| undefined` is fine with me 👍

Reply:
I think I would rather change this so that `this.translations` has the same "no translation found" type as we give when you serialize it (`null`)